### DISCUSSION TITLE
Disable email notification on backup success

### DISCUSSION THREAD

**Discussion by vinespie** [akeeba/panopticon#712] [Ideas]
It would be useful to be able to enable or disable email notifications for successful backups. When there are many sites and configured backups, a lot of emails are received. In the mass of emails, an error might be missed. By default, email notifications for successful backups should be enabled.

**Comment by david-unwin** [comment]
Agreed !

**Comment by nikosdion** [comment]
One solution is to delete the email template
* Administration, Mail Templates.
* Select the "Akeeba Backup: Successful backup" item.
* Click on Delete in the toolbar.

However, you have no indication if a backup was taken or not, which is the entire point of an email.

So, instead of removing the email altogether why don't you guys just learn to use your email client? :) Set up a mail filter which moves these emails to a different folder. You know the sender and the subject. Right? You can even set up auto-expunge on the folder i.e. auto-delete the emails after a period of time. 

If you're all fancy-pants you can even create a rule on the server itself to move emails matching a pattern to a different folder, or email account, or even delete them automatically.

Whatever you do, you should KEEP these emails for a while. When your backups fail to run you will understand why.

**Comment by vinespie** [comment]
Okay, I will test the method by disabling the email template. 

Regarding the filter in the email client, I agree, but let's think about our planet a bit; let's not send emails unless we really need to. :)

**Reply by nikosdion** [reply]
Two options will be added to _scheduled_ backups controlling whether emails will be sent on success and on failure, respectively. This will be made available in version 1.2.0.
